discovery-provider/plugins/notifications/src/web.ts

```diff
@@ -0,0 +1,82 @@
+import webpush from 'web-push'
+import { logger } from './logger'
+import { Browser, UserNotificationSettings, WebPush } from './processNotifications/mappers/userNotificationSettings'
+
+export const configureWebPush = () => {
+  // setup and configuration
+  let webPushIsConfigured = false
+  const vapidKeys = {
+      publicKey: process.env.BROWSER_PUSH_VAPID_PUBLIC_KEY,
+      privateKey: process.env.BROWSER_PUSH_VAPID_PRIVATE_KEY
+    }
+  const browserPushGCMAPIKey = process.env.BROWSER_PUSH_GCM_API_KEY
+  if (vapidKeys.publicKey && vapidKeys.privateKey && browserPushGCMAPIKey) {
+    webpush.setGCMAPIKey(browserPushGCMAPIKey)
+    webpush.setVapidDetails(
+      'mailto:contact@audius.co', // taken from identity
+      vapidKeys.publicKey,
+      vapidKeys.privateKey
+    )
+    // set config to true
+    webPushIsConfigured = true
+    logger.info('Web Push is configured')
+  } else {
+    logger.warn(
+      'Web Push is not configured. Browser Push API Notifs will NOT be sent'
+    )
+  }
+  globalThis.webPushIsConfigured = webPushIsConfigured
+  return webPushIsConfigured
+}
+
+// utility functions
+export const sendBrowserNotification = async (settings: UserNotificationSettings, userId: number, title: string, message: string) => {
+    let numSentNotifs = 0
+  
+    try {
+      if (!globalThis.webPushIsConfigured) return numSentNotifs
+      const userBrowserSettings = await settings.getUserBrowserSettings([userId])
```

Comment:
i think the browser settings can be accessed like this since it's already queried

      const userBrowserSettings = settings.browser[userId]

Reply:
i would actually ask that we move all this verification stuff (lines 37 to 47) into a function in userNotificationSettings like `getUserNotificationBrowsers(userId)` so ideally we can keep this function to just sending out the browser notifs

also did we want to add checks that these specific web push notifs were enabled before sending based on the UI? 
we had a similar problem with mobile push notifications and you can search for the fn `isNotificationTypeEnabled` to see we did extra checks for a handful of notifs.

```favorites: boolean
      milestonesAndAchievements: boolean
      reposts: boolean
      announcements: boolean
      followers: boolean
      remixes: boolean
      messages: boolean```

Reply:
I can add the checks now but we won't really be able to test until Michelles fix for persisting the settings changes goes in.